resources/assets/react/screens/Home.js

```diff
@@ -15,25 +15,87 @@ import actions from '../redux/actions';
 
 import ArticleList from '../components/Article/List';
 
-@connect(store => ({
-	articles: store.getData('articles'),
-	fetching: store.isFetching('articles'),
-	fetched: store.isFetched('articles'),
-}))
+const buttonStyle = {
+	display: 'block',
+	border: '1px solid black',
+	borderRadius: '10px',
+	margin: '0 auto 10px auto',
+	width: '150px',
+	textAlign: 'center',
+	paddingTop: '10px',
+	paddingBottom: '10px',
+};
+
+@connect()
 class ScreensHome extends React.Component {
+	constructor(props) {
+		super(props);
+		// fetched : are the first articles fetched ?
+		// page : page counter
+		// canAskMore : set to false when fetching or when there are no more articles)
+		this.state = { page: 1, fetched: false, canAskMore: false };
+		// I want this function to remember who is "this".
+		// See https://stackoverflow.com/questions/33973648/react-this-is-undefined-inside-a-component-function
+		this.moreArticles = this.moreArticles.bind(this);
+	}
+
 	componentWillMount() {
```

Comment:
ComponentWillMount is deprecated [https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html)